docs/installation.rst

```diff
@@ -124,7 +124,8 @@ We highly recommend deleting your existing environment and creating a new one us
 Use the following ``conda`` command to remove your current ``tardis`` environment:
 
 ::
-  $ conda remove --name tardis --all
+
```

Comment:
why empty line? Is this intentional?

Reply:
It wont show the command as code otherwise. 
Old:
![image](https://github.com/user-attachments/assets/9a4c648d-89ad-4e1f-babd-6b3697a4ae3e)
New:
![image](https://github.com/user-attachments/assets/c60ba910-d96e-435e-a79a-5da159332e3a)